sdk/lusid/utilities/api_configuration_loader.py

```diff
@@ -0,0 +1,55 @@
+import os
+import json
+from urllib.request import pathname2url
+from .api_configuration import ApiConfiguration
+
+
+class ApiConfigurationLoader:
+
+    @staticmethod
+    def load(api_configuration):
```

Comment:
See earlier comment about confusion of this parameter name to the ApiConfiguration class

Reply:
done